package.json

```diff
@@ -21,8 +21,14 @@
   "eslintConfig": {
     "extends": [
       "react-app",
-      "react-app/jest"
-    ]
+      "react-app/jest",
```

Comment:
Jest is a JavaScript test runner that provides resources for writing and running tests.